reference/api/rest.rst

```diff
@@ -0,0 +1,358 @@
+REST API
+========
+
+HTTP Verbs
+----------
+
+====== ===========
+Verb   Description
+====== ===========
+GET	   Used for retrieving a resource or a collection of resources.
+POST   Used for creating a resource.
+PATCH  Used for partially updating a resource.
+PUT	   Used for replacing a resource.
+DELETE Used for deleting a resource.
+====== ===========
+
+Pagination
+----------
+
+All responses containing a collection of resources will be paginated by 10 items by default.
+
+.. warning::
+    You cannot request more than 100 resources at the same time.
+    An error with code 400 will be thrown if the limit is bigger than 100.
```

Comment:
=> increase the limit. And why this limit ?
In fact, on certain case, providing the last page is even worst than applying a limit, at it requires a count, which is costly and in certa
in specific case cannot be computed precisely (see Google Search for example).